integrationTests/vm/esdt/esdtMultisign_test.go

```diff
@@ -0,0 +1,309 @@
+		proposers = append(proposers, walletAddressAsHex)
+	}
+
+	parameters := []string{
+		hex.EncodeToString(contractBytes),
+		hex.EncodeToString(vmType),
+		hex.EncodeToString(codeMetaData.ToBytes()),
+		hex.EncodeToString(big.NewInt(int64(len(proposersIndexes))).Bytes()),
+	}
+	parameters = append(parameters, proposers...)
+
+	txData := strings.Join(
+		parameters,
+		"@",
+	)
+
+	multisigContractAddress, err := nodes[ownerIdx].BlockchainHook.NewAddress(
+		nodes[ownerIdx].OwnAccount.Address,
+		nodes[ownerIdx].OwnAccount.Nonce,
+		vmType,
+	)
+	require.Nil(t, err)
+
+	log.Info("multisign contract", "address", integrationTests.TestAddressPubkeyConverter.Encode(multisigContractAddress))
+	integrationTests.CreateAndSendTransaction(nodes[ownerIdx], nodes, big.NewInt(0), emptyAddress, txData, 100000)
+
+	return multisigContractAddress
+}
+
+func proposeIssueTokenAndTransferFunds(
+	nodes []*integrationTests.TestProcessorNode,
+	multisignContractAddress []byte,
+	initalSupply *big.Int,
+	ownerIdx int,
+) {
+	tokenName := []byte("token")
+	issuePrice := big.NewInt(1000)
+	ticker := "TKN"
+	multisigParams := []string{
+		"proposeSCCall",
+		hex.EncodeToString(vm.ESDTSCAddress),
+		hex.EncodeToString(issuePrice.Bytes()),
+	}
+
+	esdtParams := []string{
+		hex.EncodeToString([]byte("issue")),
+		hex.EncodeToString(tokenName),
+		hex.EncodeToString([]byte(ticker)),
+		hex.EncodeToString(initalSupply.Bytes()),
+		hex.EncodeToString([]byte{6}),
+	}
+
+	hexEncodedTrue := hex.EncodeToString([]byte("true"))
+	tokenPropertiesParams := []string{
+		hex.EncodeToString([]byte("canFreeze")),
+		hexEncodedTrue,
+		hex.EncodeToString([]byte("canWipe")),
+		hexEncodedTrue,
+		hex.EncodeToString([]byte("canPause")),
+		hexEncodedTrue,
+		hex.EncodeToString([]byte("canMint")),
+		hexEncodedTrue,
+		hex.EncodeToString([]byte("canBurn")),
+		hexEncodedTrue,
+	}
+
+	params := append(multisigParams, esdtParams...)
+	params = append(params, tokenPropertiesParams...)
+	txData := strings.Join(params, "@")
+
+	integrationTests.CreateAndSendTransaction(nodes[ownerIdx], nodes, big.NewInt(1000000), multisignContractAddress, "deposit", 100000)
+	integrationTests.CreateAndSendTransaction(nodes[ownerIdx], nodes, big.NewInt(0), multisignContractAddress, txData, 100000)
+}
+
+func getActionID(t *testing.T, nodes []*integrationTests.TestProcessorNode, multisignContractAddress []byte) []byte {
+	node := getSameShardNode(nodes, multisignContractAddress)
+	accnt, _ := node.AccntState.LoadAccount(multisignContractAddress)
+	_ = accnt
+
+	query := &process.SCQuery{
+		ScAddress:  multisignContractAddress,
+		FuncName:   "getPendingActionFullInfo",
+		CallerAddr: make([]byte, 0),
+		CallValue:  big.NewInt(0),
+		Arguments:  make([][]byte, 0),
+	}
+
+	vmOutput, err := node.SCQueryService.ExecuteQuery(query)
+	require.Nil(t, err)
+	require.Equal(t, 1, len(vmOutput.ReturnData))
+
+	actionFullInfo := vmOutput.ReturnData[0]
+
+	return actionFullInfo[:4]
```

Comment:
Why [:4] ?

Reply:
actionFullInfo contains the full byte slice containing all the action IDs and detailed information. The action ID is stored in the first 4 bytes.

Reply:
ok